site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
+date: 2020-03-09T11:35:32
+weight: 3
+subsection: "Setup Push Notifications"
```

Comment:
```suggestion
subsection: "Set Up Push Notifications"
```

Reply:
@justinegeffen Did we yet implement a change in our guidelines to use sentence case in headings?

Reply:
I've just added it to the PR.